README.adoc

```diff
@@ -93,15 +93,15 @@ name-deployment-6bd97d9bf6-6d2cj   1/1       Running   0          34s
 ping-deployment-645767664f-7gnxf   1/1       Running   0          34s
 ----
 
-Then navigate to `{name-api}` and use the username `bob` and the password `bobpwd` to authenticate. Replace `[hostname]` with the ip address or hostname of your {kube} cluster. By default it will be `{minikube-ip}` for MiniKube and `localhost` for Docker Desktop. You will see something similar to `Hello! I'm container name-deployment-6bd97d9bf6-qbhbc`.
+After the pods are running, navigate to `{name-api}` and use the username `bob` and the password `bobpwd` to authenticate. Replace `[hostname]` with the IP address or hostname of your {kube} cluster. By default, it will be `{minikube-ip}` for MiniKube and `localhost` for Docker Desktop. You will see something similar to `Hello! I'm container name-deployment-6bd97d9bf6-qbhbc`.
 
 Similarly, navigate to `{ping-api}/name-service` and you will see `pong`.
 
 == Modify name application
 
 First, update `name` to have a configurable greeting message. At the moment, it is hardcoded to be `Hello!`. Use MicroProfile Config and CDI to inject the value of an environment variable called `GREETING` into the `greeting` member of the `NameResource` class. Then use `greeting` to create a response with a custom greeting message.
 
-Update the name resource class in the `name/src/main/java/io/openliberty/guides/name/NameResource.java` file:
+Update the name resource class in the `name/src/main/java/io/openliberty/guides/name/NameResource.java` file.
```

Comment:
Generally, we use the ":" instead of "." for such class creation/update statement. If we make the change for this guide, it's inconsistent with the other guides. @helyarp which one is the best recommendation?

Reply:
Your choice. If you usually use colon (:) for the guides, then colon is prefereable.

Reply:
@proubatsis ^

Reply:
Fixed @evelinec